src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -4582,7 +4582,7 @@ internal string FixupFileName(string moduleBase, string name, string extension)
             if (!string.IsNullOrEmpty(ext) && ext.Equals(".dll", StringComparison.OrdinalIgnoreCase))
             {
                 Exception ignored = null;
-                Assembly assembly = ExecutionContext.LoadAssembly(name, null, out ignored);
+                Assembly assembly = ExecutionContext.LoadAssembly(name, result, out ignored);
```

Comment:
If we already got the resolved assembly path, we can skip the loading, right? The loading is solely for getting the path of the assembly. When the target assembly is in GAC, we have to load it in order to get the path, but not if we already resolved the assembly file path.

Reply:
In this case, we actually want to load from module base first. In the `Resolve` method, we do not have access to module base so we need to do this here. If there are alternatives, please suggest.

Reply:
Please add a comment to the code so it is clear to other developers that we really want to load the assembly here, not just return the pathname.  Thanks!